DistributionPackages/Neos.Presentation/Resources/Private/Fusion/Atom/Interactive/Interactive.pcss

```diff
@@ -21,3 +21,27 @@
     @apply border-gray hover:border-gray;
   }
 }
+
+.n-interactive--tag {
+  @apply inline-block display-body-md;
```

Comment:
`@apply display-body-md;`
this does not work, as it is not a class defined by tailwind